lib/http-routes/api/create-call.js

```diff
@@ -140,6 +141,16 @@ router.post('/',
         }
       }
 
+      // find handling sbc sip for called user
+      if (target.type === 'user') {
+        const { registrar }  = srf.locals.dbHelpers;
+        const reg = await registrar.query(target.name);
+        if (reg) {
+          sbcAddress = selectHostPort(logger, reg.sbcAddress, 'tcp')[1];
```

Comment:
Hmm, I am not loving this.  Feature server already selects an SBC to send to (prior to this change) and I believe it finds the IP address of the SBC in the VPC (the private address).  How is that done?  If different than this, why?

Reply:
@davehorton 

feature server look at redis for keys default:active-sip to get list of sbc private ip address.

then select next sbc address from the list to send request to. this way, feature server cannot send INVITE request to correct sbc which is handling for the user.

this code will get registration from redis for the user, get sbc_address which is handling for the user and as you know that there is a tcp interface from drachtio server will contains private IP address for the sbc. feature server use this private IP to send the request to that SBC.

Reply:
its ugly but for now I'll accept.  We need to come back and fix this at some point -- simplify the learning of sbc private/internal address